cves/kernel/CVE-2019-15666.yml

```diff
@@ -167,8 +167,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    A series of unit tests could have easily been developed to test the edge
+    cases of this array search. There also could have been try/catch statements
```

Comment:
The C language doesn't have try/catch - you mean using a conditional to check if the index is out of bounds?